bytelatent/data/patcher.py

```diff
@@ -24,8 +26,12 @@ class PatchingModeEnum(str, Enum):
 
 
 class PatcherArgs(BaseModel):
+    class Config:
+        arbitrary_types_allowed = True
+
     patching_mode: PatchingModeEnum = PatchingModeEnum.entropy
     patching_device: str = "cuda"
+    entropy_model: nn.Module | None = None
```

Comment:
The configs here are suppose to be serializable and read from a file, so this would break this. I'm definitely open to having a way to set this more flexibly, but I don't think it should be in `PatcherArgs`.

I would instead suggest that you have a variable/flag here that is serializable, then create another function that can take those variables/flags to construct the `nn.Module` you supply here.